src/hooks/result.inc

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+set -e
+
+# Check the execution of some command is ok
+# $1: Returned code by process
+execution_success() {
+
+    if [ "$#" -eq 1 ]
+    then
+        if [ -z "${_shunit_test_}" ]
+        then
+            test_name="Execution"
+        else
+            test_name="Execution of test '${_shunit_test_}'"
+        fi
+
+        assertEquals "$test_name should retrieve exit code 0" "0" "$1"
+        return 0
+    else
+        echo "Error. Missing parameters:"
+        echo "   Returned code by process"
+        return 1
+    fi
+
+}
+
+# Check the execution message of some command contains an expected pattern
+# $1: Returned message
+# $2: Expected contained value
+message_contains() {
+
+    if [ "$#" -eq 2 ]
+    then
+        if [ -z "${_shunit_test_}" ]
+        then
+            test_name="Message from execution"
+        else
+            test_name="Message from execution of test '${_shunit_test_}'"
+        fi
+
+        assertContains "$test_name should contain '$2'" "$1" "$2"
```

Comment:
So the idea is that you have obtained the message somehow and you use this utility function to verify the content right?

Reply:
you're right!